### ISSUE TITLE
Various DX11 Games Suffer From Intense Stuttering, Borderline Unplayable w/o DXVK

### ISSUE THREAD

**Issue by JohnnyTex** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#247]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Game is not cracked, modded and use the latest patch

### Game / Application [Required]

Various DX 11 Games (Tested in Sonic Frontiers Most Recently)

### Game Platform [Required]

- [X] Steam
- [ ] Microsoft Store
- [ ] GOG Galaxy
- [ ] Ubisoft Connect
- [ ] Battle.net
- [ ] Epic Games
- [ ] Rockstar Games Launcher
- [ ] EA app
- [ ] Other (please specify)

### Processor / Processor Number [Required]

i5 12600k

### Graphic Card [Required]

ARC A770 LE

### GPU Driver Version [Required]

- [X] 31.0.101.4123
- [ ] 31.0.101.4091
- [ ] 31.0.101.4090
- [ ] 31.0.101.4034
- [ ] 31.0.101.4032
- [ ] 31.0.101.3975
- [ ] 31.0.101.3959
- [ ] 31.0.101.3802
- [ ] 31.0.101.3793
- [ ] 31.0.101.3491
- [ ] 31.0.101.3490

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [ ] DirectX12
- [X] DirectX11
- [ ] DirectX10
- [ ] DirectX9

### Windows Build Number [Required]

- [X] Windows 11 22H2
- [ ] Windows 11 21H2
- [ ] Windows 10 22H2
- [ ] Windows 10 21H2
- [ ] Other (Please specify)

### Intel System Support Utility report

[4123ssu.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/10752928/4123ssu.txt)

### Description and steps to reproduce [Required]

It's as the title says. Just moving the camera left and right a little bit is enough to make the game almost look like it's about to freeze. The game chugs as it goes forward, and on top of that fps plunges are common in Sonic Frontiers. For example, once you get to the main island in Sonic Frontiers (after you collect the first Chaos Emerald if I recall correctly), looking at the center of the field where the optional boss is located drags the fps down to the 20fps range. When using DXVK, these issues are often totally removed, and the game runs at a silky smooth 60fps. 

### Game graphic quality [Required]

- [ ] Low
- [ ] Medium
- [X] High
- [ ] Ultra

### Game resolution [Required]

2560 x 1440

### Game VSync [Required]

Off

### Game display mode [Required]

- [X] Fullscreen
- [ ] Windowed
- [ ] Borderless Windowed

### Detailed game settings [Required]

The games in question run at high presets.

**Comment by JohnnyTex** [comment]
I should note that even though DXVK 2.1 improves Sonic Frontiers' performance tenfold, the game still has stuttering issues, especially when booting into the game world for the first time.

**Comment by IntelSupport-Rozilah** [comment]
hi @JohnnyTex,
Thanks for contacting Intel, we will start verifying the issue once the system is ready and stay tuned!.

**Comment by ABagOfPotato** [comment]
Hi,

Just want to add to this because I don't think it warrants opening another thread given that this issue is for various DX11 games.

Dead By Daylight is currently having severe frame drops, I thought it was better a few drivers ago but having rolled back I'm still experiencing frame drops down into the mid 30's down from 90-95fps, making it actually really difficult to play the game. I've had to switch back to my RX 6600 which has no issue with the game at 1440p at a mostly locked 95hz/fps.

Worth adding here because it doesn't look like the game should be performing that badly, and in the main menus it works fine FPS wise. Also worth mentioning as it's still a very popular multiplayer title.

Hopefully this gets addressed along with the others. If I should make a separate thread about it please let me know and I can do!

**Comment by JohnnyTex** [comment]
Specifically in your case, I'd recommend running DBD in DX12. I play that game at ultra settings, 1440p, and I get a largely stable 120fps, w/ frame drops into the 100s.

**Comment by ABagOfPotato** [comment]
> Specifically in your case, I'd recommend running DBD in DX12. I play that game at ultra settings, 1440p, and I get a largely stable 120fps, w/ frame drops into the 100s.

Thanks for the tip! I've given it a go quickly and yep, it requires setting a launch command in steam manually but it makes it run in DX12, apart from some shader compile stutter to start with its then at a locked 95fps for me!(my refresh rate).

Hopefully they can fix this DX11 issue though still, as DBD tend to take things away when it comes onto their radar! (Back when DbD started working on Proton and Steam deck for like 3 weeks straight they found out and then they specifically blocked it so that it would kick players out of matches if they were on linux).

But yeah, for the time being its a good call! Unfortunately there will be lots of people having my experience though and not knowing what to do, so it needs to be looked at overall if they want to get to average end users.

**Comment by Arturo-Intel** [comment]
Hello @JohnnyTex,
Did you try to reproduce this behavior with the [31.0.101.4314](https://www.intel.com/content/www/us/en/download/726609/intel-arc-iris-xe-graphics-whql-windows.html?)? 

-- r2

**Comment by JohnnyTex** [comment]
I believe I did on 4314, but now that we're on 4335 I decided to go back and give it a try anyways. And lo and behold, Sonic Frontiers' framerate seems to be fixed by and large! There's still the issue of its fullscreen mode breaking adaptive sync, but maybe that's a separate issue. I'll go around and take a look at how other DX11 games are performing, and with any luck these improvements are across the board.

**Comment by JohnnyTex** [comment]
There might be issues w/ DX11 games and fullscreen. When I tried to load up Assassin's Creed Unity for further testing it was a struggle to even get the game going because in fullscreen the monitor kept freaking out. Had to switch to borderless.

**Comment by Arturo-Intel** [comment]
@JohnnyTex I see, glad to see the FPS is right where it has to be
Can you please open another case for the full screen problem? 

And I think we can close this case right?

**Comment by JohnnyTex** [comment]
Not yet, this wasn't just specific to Sonic Frontiers. And I already have another ticket to tackle this specific example, but AC Unity isn't quite there yet when it comes to fps. And I haven't tested this in other DX11 games that suffered from this. I think this weekend will be my chance to do some more testing, as I'll have a break in my finals schedule then.

**Comment by Arturo-Intel** [comment]
OK, I will be waiting then
GL on your finals!

**Comment by JohnnyTex** [comment]
I had a bit of time to squeeze in some further testing. None of this is definitive, but here are the results so far in the following games;
Sonic Frontiers is improved immensely, but fps drops are still frequent. Nowhere near as bad, but drops from 60fps down to 45 aren't what I'd call a good place to be in. 
Sonic Origins' core gameplay seems good, but the menus (which are the 3d-rendered parts) are very laggy, with frequent drops down to the 30s in fps. 
Assassin's Creed Unity actually probably had DXVK on in the background (I had previously tried using DXVK to restore compatibility to the game). I deleted the DXVK files to test its raw performance, and tragically without DXVK the game is still unplayable (20fps). With DXVK, it can get up to 75fps. This is technically a slight improvement, as before this driver AC Unity didn't work w/ DXVK. I'm gonna have to update the AC Unity ticket with this new info. 
TL;DR, the preliminary results are that DX11 performance has gone up, but is still not truly competent yet in the games that saw improvements. And games like AC Unity have seen no meaningful improvement.

**Comment by Arturo-Intel** [comment]
@JohnnyTex The latest driver (4514) improves a lot of dx11 games, can you try it on this case?

NOTE: DbD issue is being worked right now more info in #288.

**Comment by JohnnyTex** [comment]
Holy, sorry for not posting updates. On Assassin's Creed Unity, the implemented changes are absolutely in full force, and I'd say at QHD High the game is a 60fps average, tending to go above more than below. On a sidenote, what's truly exciting about that is the fact it's a native solution! Not using DXVK, which can cause issues w/ things like Auto HDR. I hope it's a sign of good things to come on the DX11 front! On Valorant, I _did_ see an increase in fps, but that's probably bc of the fact that I have a new processor (13700). A770 still has perplexing slow spots in that game though. I'll admit, I haven't done too much testing on it because that game makes me want to throw my mouse through a wall. Sonic Origins still exhibits extreme lag on the menus, but the gameplay is still totally fine. An odd case overall that one. I haven't played very much of Sonic Frontiers and Tales of Arise due to the aforementioned performance issues, but here's just some quick observations; Sonic Frontiers seems to have had its slowdowns reduced even further. This week I'll try and have some more updates on it, but it seems like it's very nearly there, though fps drops down to the 40s still happen, even if it's briefly. Tales of Arise I haven't touched since, but I scarcely remember it being playable even without. I'll have more to say about it in the coming days.

**Comment by JohnnyTex** [comment]
I'll note that Splitgate, another DX11 game, runs very well, often exceeding 200fps at QHD high. It's got its own compatibility issues, but in regards to performance it's there. I'm not sure how relevant that game's performance is, as it's not terribly demanding, but it's a good showcase. I'll take a look later this week to see if there are any other DX11 games I have or are about to buy, and will report on their performance.

**Comment by scottrick** [comment]
I'm running driver 4644 on my Intel A770 LE.  I am playing Pillars of Eternity 2 at 1440p, and the game stutters very poorly.  I had played a couple months ago and it was much better, so not sure what happened.  Changing the graphical settings from the very lowest to the highest doesn't seem to make any difference.

**Comment by freak2fast4u** [comment]
@ABagOfPotato , @JohnnyTex , @scottrick : Is this still true with the 4826 beta drivers recently released ?

**Comment by scottrick** [comment]
I'm on 4824.  I eventually got it running better by completely uninstalling my drivers with DDU and doing a clean install.

**Comment by JohnnyTex** [comment]
I'll have to run through the whole suite again to see. I played a thing of Splitgate and the previously reported issues didn't show up, but it was just a match, so I'll have to play it some more before I say Splitgate has been fixed. The other games I haven't played in a while, I'll have to get back into them.

**Comment by JohnnyTex** [comment]
Valorant's issues have been resolved but that was seemingly due to an Intel-targeted update on their end, so that's that.

**Comment by JohnnyTex** [comment]
I just did a quick test in Sonic Origins, and it seems like that game has been freed from the insane stuttering in the menus as well. That was a big one.

**Comment by JohnnyTex** [comment]
Sonic Frontiers has seemingly been fixed too! More in-depth testing will have to be done, but I deleted the DXVK files I had in them previously and the games seem to run very well w/o them. Dead by Daylight continues to crash when playing in DX11, but I have another update to make on that game on its own thread. I'll do more testing hopefully over this Fall Break.

**Comment by JohnnyTex** [comment]
On one hand, Fortnite DX11 is still not desirable, performing worse than DX12, which means the Performance Mode isn't something ARC users can benefit from. But so far, it seems to me that in an odd twist, COD Cold War performs just as well if not slightly better in DX11 than 12. idk if I can prove that in a repeatable way, but yeah. Quite impressive.

**Comment by JohnnyTex** [comment]
I wanna avoid mission creep here; DX11 support on ARC probably still isn't as good as what can be accomplished w/ DXVK, but if the results are at least competitive I'll consider it mission accomplished.

**Comment by rmorera-intel** [comment]
Hi @JohnnyTex as you probably saw we continue to roll-out drivers to improve performance on several DirectX 11 titles. For Assassins Creed Unity the uplift in performance should be since driver 101.4514 (and newer).

I'll close this one now.
